pom.xml

```diff
@@ -165,18 +165,20 @@
                         <rulesets>
                             <ruleset>${project.parent.parent.basedir}/pmdrules.xml</ruleset>
                         </rulesets>
+                        <analysisCache>true</analysisCache>
```

Comment:
Nevermind, the state introduced with this can cause a build failure when switching branches.  I would rather have consistent builds than shave a second or two off of the build.